Kernel/FileSystem/Ext2FileSystem.cpp

```diff
@@ -454,6 +454,8 @@ Vector<Ext2FS::BlockIndex> Ext2FS::block_list_for_inode_impl(const ext2_inode& e
     if (!blocks_remaining)
         return list;
 
+    // Don't need to make copy of add_block, since this capture will only
+    // be called before block_list_for_inode_impl finishes.
```

Comment:
This comment seems kind of excessive to me. Are we supposed to put these whenever we use lambdas that don't persist past the end of a function?

Reply:
My thought process was: "It took me quite a while to verify that it's correct, let me just put a comment so the next person can verify it more quickly."

If you'd like me to remove the comment, I can.

Reply:
Nah, it's all good. Just commenting :)